config.yaml

```diff
@@ -41,7 +41,7 @@ build_iso:
     mock_args:
     packages_dir: "result/packages/latest"
     automated_install_file: "host-os.ks"
-    host_os_packages_groups:
+    installable_environments:
```

Comment:
Those environments do not have to contain Host OS packages. They do have CentOS groups too, coming from the `minimal_install_groups` option.
If, for some reason, someone asks us to include an installation option to install a set of CentOS packages from extras, or whatever, this could be used.